shaders/glsl/texture_ccolor.fragment

```diff
@@ -23,13 +23,12 @@ varying vec3 color;
 LAYOUT_BINDING(0) uniform sampler2D TEXTURE_0;
 
 
-//
-/////////////////////// some UI, transition between dimensions texture and skybox shader ////////////////////////
+///////////////////////Some UI, transition between dimensions, texture, and skybox shaders////////////////////////

```

Comment:
Again. There is only one shader but for some things

Reply:
I'll change this